libs/entry-components/.eslintrc.json

```diff
@@ -31,7 +31,8 @@
             "prefix": "entry",
             "style": "kebab-case"
           }
-        ]
+        ],
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
Why did you turn off this one?

Reply:
We are using any in more then few places in the library.
Maybe there is good justification for most of them, we can investigate this in a separate issue if you agree.

e.g.
https://github.com/enigmatry/entry-angular-building-blocks/blob/adfea80b1557f0be59bedaa09b392a92d0eac41f/libs/entry-components/dialog/dialogs/entry-dialog.component.ts#L38

https://github.com/enigmatry/entry-angular-building-blocks/blob/adfea80b1557f0be59bedaa09b392a92d0eac41f/libs/entry-components/dialog/entry-dialog.service.ts#L69